modules/dxp/apps/segments/segments-experiment-web/test/js/components/ClickGoalPicker.es.js

```diff
@@ -96,28 +100,27 @@ describe('Segments Experiments with Click Goal', () => {
 		);
 
 		expect(experiment.status.label).toBe('Draft');
-		const editElement = getByText('edit-element');
+		const editElement = getByText('change-clickable-element');
 		expect(editElement.attributes['disabled']).toBe(undefined);
 	});
 
-	it('When the user set element as click target, then the element is set as the click target and the id of the element as a link', () => {
+	it('When the user select element as click target, then the element is select as the click target and the id of the element displays inside the input', () => {
```

Comment:
I find this a bit hard to read (not fully grammatical). I don't even know what edit to suggest.

Reply:
Maybe something like: "The user selects a clickable element target and the HTML id of the element displays inside the input". I'm taking a look to all the test names 🧐